CHANGELOG.rst

```diff
@@ -18,6 +18,7 @@ Improvements and new features:
  - Early fail when output cannot be created. (:issue:`346`)
  - Add option :option:`--txt` for text output. (:issue:`387`)
  - Add option :option:`--csv` for CSV output. (:issue:`376`)
+ - Add option :option:`--coveralls` for coveralls JSON output. (:issue:`328`)
```

Comment:
Here can be a conflict after my documentation updates, so maybe you should rebase this branch on the current master.